pkg/os/util.go

```diff
@@ -115,3 +115,7 @@ func RemoveFileIfExists(path string) error {
 func RunningInTerminal() bool {
 	return terminal.IsTerminal(int(os.Stdin.Fd()))
 }
+
+func RunningUsingSSH() bool {
+	return os.Getenv("SSH_TTY") != ""
```

Comment:
Or `LookupEnv`? I guess `SSH_TTY` is not going to be set but empty, but `LookupEnv` seems more correct

Reply:
@cfergeau It is going to be set
```
// On my current workstation
$ env | grep -i ssh
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
SSH_AGENT_PID=5205


// On the remote machine
$ env | grep -i ssh
SSH_CONNECTION=10.10.113.37 45514 10.0.78.252 22
SSH_CLIENT=10.10.113.37 45514 22
SSH_TTY=/dev/pts/0
```